pkg/lint/fileparser.go

```diff
@@ -19,7 +19,7 @@ import (
 var (
 	fMap                 = dummyFuncMap()
 	includes             = &includeExtract{[]string{}}
-	templatePathPattern  = regexp.MustCompile(`(\.[a-zA-Z].[a-zA-Z0-9_\.]+)`)
+	templatePathPattern  = regexp.MustCompile(`(?:\{\{.*?[\-\s\(]?)(\.[a-zA-Z][a-zA-Z0-9_\.]+)(?:[\-\s\)]?.*?\}\})`)
```

Comment:
Matched pattern must fulfill the following conditions:
- be found between template delimiters `{{` & `}}`
- might be delimited by spaces, `{{-` & `-}}`, or `()` (in case of conditionals)
- has to start with a letter

Reply:
![it-is-magic-5b4cb3](https://user-images.githubusercontent.com/1071648/129181361-5ddc3d38-5f8e-427c-8059-0676d621c4c3.jpg)